### ISSUE TITLE
Errors out installing Monumental Experience mod pack

### ISSUE THREAD

**Issue by monkeyman512** [cdbbnnyCode/modpack-installer#18]
I am trying to install modpack [Monumental Experience-17.0.61.zip](https://www.curseforge.com/minecraft/modpacks/monumental-experience/files/3949216)

 The program errors out with this message:
```
Linking mods
Extracting texture pack .modcache/STONEBORN Gear-HIGH_EYES-1.16.5 - V2.1.zip
Extracting texture pack .modcache/EmbellishedStone.zip
Extracting texture pack .modcache/STONEBORN - 1.16.5 - V3.1.zip
Extracting texture pack .modcache/Repurposed_Structures-More_Villagers_Forge_v2.zip
Traceback (most recent call last):
  File "/home/<user>/github_public/modpack-installer/install.py", line 244, in <module>
    main(args.zipfile, args.mcdir, args.forge_disable)
  File "/home/<user>/github_public/modpack-installer/install.py", line 208, in main
    for dir in os.listdir(texpack_dir + '/assets'):
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/334827/assets'
```

**Comment by monkeyman512** [comment]
Looks like the problem is that the code is assuming the mod zip will have an "assets" dir in it, but this mod does not:
![image](https://user-images.githubusercontent.com/3687204/187060782-4ca707bb-762b-4839-a04c-89e7cb62c6ad.png)

**Comment by cdbbnnyCode** [comment]
oh this is a data pack, I assume? My code assumes anything marked as a "texture pack" is a resource pack with an assets/ folder. This should be a fairly simple fix, though.

**Comment by monkeyman512** [comment]
I made a pull request with the fix: https://github.com/cdbbnnyCode/modpack-installer/pull/19

**Comment by cdbbnnyCode** [comment]
Update - I have made a fix that works for the Repurposed Structures datapacks and should work for other datapacks as well. The issue was that in vanilla MC, there doesn't seem to be a way to install a datapack globally for all new worlds (it has to be installed manually in the 'Create New World' menu). However, it seems like putting the datapacks into '.minecraft/datapacks' does allow them to be picked up automatically, at least in the one modpack I tested.

TLDR: Fixed in 65263b6 and 6fe8c76
